code/datums/components/storage/concrete/wallet.dm

```diff
@@ -1,5 +1,5 @@
 /datum/component/storage/concrete/wallet/on_alt_click(datum/source, mob/user)
-	if(!isliving(user) || !user.CanReach(parent))
+	if(!isliving(user) || !user.CanReach(parent) || (user.stat == SOFT_CRIT || UNCONSCIOUS))
```

Comment:
if(user.incapacitated()) instead

also, update the parent proc so the check is in the same location

Reply:
Good call, I didn't know that function existed.